contracts/strategies/stkCvxFXS/stkCvxFxsVault.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "../../GenericVault.sol";
+
+struct EarnedData {
+    address token;
+    uint256 amount;
+}
+
+interface stkCvxFxsStrategy {
+    function harvest(address _caller, uint256 _minAmountOut)
+        external
+        returns (uint256 harvested);
+}
+
+interface ICvxFxsStaking {
+    function claimableRewards(address _account)
+        external
+        view
+        returns (EarnedData[] memory userRewards);
+}
+
+contract stkCvxFxsVault is GenericUnionVault {
+    bool public isHarvestPermissioned = false;
```

Comment:
Can be simplified to `bool public isHarvestPermissioned;` since the default value is already `false`.